packages/itwin/property-grid/src/PropertyGridManager.ts

```diff
@@ -8,22 +8,34 @@ import { IModelApp } from "@itwin/core-frontend";
 import type { Localization } from "@itwin/core-common";
 import type { LocalizationOptions } from "@itwin/core-i18n";
 
+/** Supported Feature Flags for  */
+export interface PropertyGridFeatureFlags {
```

Comment:
wondering if this should be a prop in the property grid component instead of making it apart of the PropertyGridManager

Reply:
Done, a new prop was added enableAncestorNavigation